transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,462 @@
+import random
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+# defaults
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+
+# intialise
+try:
+    conjugate_en(verb='testing',tense='present',number='singular')
+except:
+    pass
+
+
+class NegateStrengthen(SentenceAndTargetOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en"]
+    tgt_languages = ["en"]
+
+    def __init__(self, seed=0, verbose=False):
+        super().__init__(seed)
+
+        # downloads
+        nltk.download('wordnet')
+        nltk.download('averaged_perceptron_tagger')
+
+        # initialise
+        self.verbose = verbose
+        self.nlp = spacy.load("en_core_web_sm")
```

Comment:
We are now using spacy 3.0.0 and loading it only once. So maybe this line can be changed to something like:

self.nlp = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")

'spacy_nlp' is present in 'initialize' file.

Reply:
How should "spacy_nlp" be loaded? Is it part of __init__()? Should the default be None?

Reply:
Spacy is loaded only once in the 'initialize' file using 'initialize_models'. You need to import 'spacy_nlp' from the 'initialize' file, its global variable. The default will be None, and in that case, will load. 

You can refer to 'ChangeTwoWayNe' or any other transformation where spacy been used.

Reply:
Ok, got it! Added and thanks!

Reply:
Based on the 'spacy_nlp' variable will load spacy, if spacy_nlp is None then load it else use already loaded spacy.

Please help to change this line: 
self.nlp = spacy.load("en_core_web_sm")
with
self.nlp = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")

Reply:
Ok, done!

Reply:
Hi! I just refactored the codes, and the tests are ok. 
In short, I realised the global spacy version (after changing this line) was tagging the sentence slightly differently from my local spacy one. I tried to use only spacy for POS tags, but the results are poor. Hence I am sticking to NLTK, but needed to realign the tokens with spacy's dependency parser.

Reply:
Might be because of a version mismatch, as we upgraded the Spacy version to 3.0.0. Did we try the same version locally?